Source/BuildControl.cs

```diff
@@ -1036,8 +1042,17 @@ public CostReport getBuildCost (ConfigNode craft, string craftText = null)
 			craftVessel.vesselName = "EL craftVessel - " + craft.GetValue ("ship");
 			craftVessel.Initialize (true);
 			foreach (Part part in craftVessel.parts) {
-				part.ModulesOnStart ();
+				// Remove module that triggers OrbitDriver and cause temporary vessel being despawned due to illegal orbit parameters
+				if (part.Modules.Contains("kOSProcessor")) {
+					var kos = part.Modules["kOSProcessor"];
+					if (kos != null) {
+						part.Modules.Remove(kos);
+					}
+				}
+
+				part.ModulesOnStart();
 			}
```

Comment:
This seems reasonable, though I think I'd prefer if it was in a separate method.

Reply:
Yes, I agree. Initially, I was thinking that this might be something more generic with the list of offending modules taken from the config or something. However, based on my checks on actual game with about 100 (mostly popular) mods installed, it seems that only kOS introduces some side-effects on module initialization.

I'll extract this into separate method, for example, SanitizePart(Part part), if you are okay with this name

Reply:
I'm certainly ok with the name for now (it can always be tweaked later). I think just having it in a separate method will go a long way to making things easier should other similar methods be needed: then they could be added to a list that needs to be iterated (ie, a modular approach to "sanitizing" parts).